docker/jenkins/run_functional_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -xe
+
+cp docker/dockerfiles/bedrock_functional_tests Dockerfile
+docker build -t bedrock_functional_tests:$GIT_COMMIT --pull=true .
+
+docker pull selenium/hub:2.48.2
+docker pull selenium/node-firefox:2.48.2
+
+docker run -d --name bedrock-code-${BUILD_NUMBER} -e ALLOWED_HOSTS="*" -e SECRET_KEY=foo -e DEBUG=False -e DATABASE_URL=sqlite:////tmp/temp.db mozorg/bedrock_code:$GIT_COMMIT
+docker run -d --name bedrock-selenium-hub-${BUILD_NUMBER} selenium/hub:2.48.2
+docker run -d --name bedrock-selenium-node-${BUILD_NUMBER} --link bedrock-selenium-hub-${BUILD_NUMBER}:hub --link bedrock-code-${BUILD_NUMBER}:bedrock selenium/node-firefox:2.48.2
```

Comment:
We can probably afford to spin up a few nodes (perhaps 5?) so tests can run in parallel. We could also do this via Docker compose to start the various containers and use [scale](https://docs.docker.com/compose/reference/scale/) for the nodes.